### ISSUE TITLE
Duplication through quote misattribution: Lao Tzu or Richard Braunstein

### ISSUE THREAD

**Issue by pscheid92** [dwyl/quotes#44]
Searching the `quotes.json` file, I detected a duplicate quote attributed to two different authors.\
As discussed in issue #41, we should check the attribution and remove the duplicated entry.

```
{
    "author": "Lao Tzu",
    "text": "He who obtains has little. He who scatters has much."
}
```
```
{
    "author": "Richard Braunstein",
    "text": "He who obtains has little. He who scatters has much."
}
```

**Comment by nelsonic** [comment]
Yeah, I think this one is a bit easier to make a call on: give it to Lau Tsu
https://www.google.com/search?q=He+who+obtains+has+little.+He+who+scatters+has+much
The first two pages of Google results attribute the quote to him.
![DDA157E6-CAE5-4136-A811-68F7BCBF18FA](https://user-images.githubusercontent.com/194400/107890777-93851900-6f12-11eb-9e3d-2479ab62f63a.png)

**Comment by pscheid92** [comment]
Agree! I'm going to open a PR removing the Richard Braunstein quote.

I found a painting using this quote as a title: [Link](https://americanart.si.edu/artwork/he-who-obtains-has-little-he-who-scatters-has-much-lao-tzu-6th-century-bc-chinese-moralist)

> “He who obtains has little. He who scatters has much.”–Lao-Tzu, 6th century B.C. Chinese moralist and mystic, on Generosity. From the series Great Ideas.

But I cannot find a single clue who Richard Braunstein is or was. I found [a notebook on Amazon using](https://www.amazon.de/who-obtains-little-scatters-much/dp/B085KBRTQD) the name and quote on the cover, but nothing more.
